Python/datacube.py

```diff
@@ -0,0 +1,41 @@
+import glob
+
+import numpy as np
+import xarray as xr
+
+def read_data(data_path, dv_key_name):
+    """Load an ensemble of CanRCM4
+    models into a single datacube.
+    ------------------------------
+    Args:
+        data_path (Str): path containing CanRCM4 ensemble
+        dv_key_name (Str): name of the design value as 
+            in the netCDF file
+    Returns:
+        ds (xarray Dataset): data cube of assembled ensemble models 
+            into a single variable
+    """
+    if not isinstance(data_path, str):
+        raise TypeError("data_path must be a string, got {}"
+                        .format(type(data_path)))
+
+    if not isinstance(dv_key_name, str):
+        raise TypeError("dv_key_name must be a string, got {}"
+                        .format(type(dv_key_name)))
+
+    nc_list = np.asarray(glob.glob(PATH+"*.nc"))
+    xr.open_dataset(nc_list[0])
+
+    xr_list = []
```

Comment:
@nikola-rados  I nuked the old method and also accidentally nuked your comment. How does this look instead? How is the readability?

Reply:
:+1: